library/Opus/Enrichment/AbstractType.php

```diff
@@ -172,6 +172,11 @@ public function setOptionsFromString($string)
         return null;
     }
 
+    public function applyValidation()
```

Comment:
Ich finde den Funktionsnamen ungünstig und würde mit hier einen Funktionskommentar wünsche (bei anderen Funktion im Prinzip auch), die die Absichten erläutern, die wir mit diesen Funktionen verbinden. Das spart in drei Jahren Zeit oder auch jetzt, wenn ein Student mit dem Code arbeiten soll.

Der Name "applyValidation" drückt eine Aktivität aus. Es geht aber darum zu fragen, ob die Validierung angewendet werden soll. Wie wäre es mit "isValidationEnabled" oder "isPerformValidation" oder "isStrictValidation"? Ich glaube das Entscheidende ist das "is" am Anfang, um eine ja/nein-Frage auszudrücken. Vielen Dank!